docs/install/README.md

```diff
@@ -35,27 +35,16 @@ to see if your system is capable of running Kata Containers.
-|[CentOS](centos-installation-guide.md)                           | 7                                                                                                                 |
-|[Debian](debian-installation-guide.md)                           | 9, 10                                                                                                             |
-|[Fedora](fedora-installation-guide.md)                           | 28, 29, 30                                                                                                        |
-|[openSUSE](opensuse-installation-guide.md)                       | [Leap](opensuse-leap-installation-guide.md) (15, 15.1)<br>[Tumbleweed](opensuse-tumbleweed-installation-guide.md) |
-|[Red Hat Enterprise Linux (RHEL)](rhel-installation-guide.md)    | 7                                                                                                                 |
-|[SUSE Linux Enterprise Server (SLES)](sles-installation-guide.md)| SLES 12 SP3                                                                                                       |
-|[Ubuntu](ubuntu-installation-guide.md)                           | 16.04, 18.04                                                                                                      |
-
 #### Official packages
 
 Kata packages are provided by official distribution repositories for:
 
-|Distribution (link to packages)                                  | Versions   |
-|-----------------------------------------------------------------|------------|
-|[openSUSE](https://software.opensuse.org/package/katacontainers) | Tumbleweed |
+|Distribution (link to packages)                          | Versions                                                                       | Contacts                                              |
+|---------------------------------------------------------|--------------------------------------------------------------------------------|-------------------------------------------------------|
+|[CentOS](centos-installation-guide.md)                   | 8                                                                              |                                                       |
+|[Fedora](fedora-installation-guide.md)                   | 32, Rawhide                                                                    |                                                       |
+|[SUSE Linux Enterprise (SLE)](sle-installation-guide.md) | SLE 15 SP1, 15 SP2                                                             |                                                       |
+|[openSUSE](opensuse-installation-guide.md)               | [Leap 15.1](opensuse-leap-15.1-installation-guide.md)<br>Leap 15.2, Tumbleweed |                                                       |
```

Comment:
Where do we build these packages now? Are they built by distribution package maintainers?

Reply:
We don't build those anymore. They're built by the distro packages maintainer.